app/views/surveys/_survey_definitions.html.erb

```diff
@@ -0,0 +1,54 @@
+<script type="application/json" data-survey-default>
+  {
+    "url": "https://www.surveymonkey.com/s/2MRDLTW?c={{currentPath}}",
+    "identifier": "user_satisfaction_survey",
+    "frequency": 50,
+    "surveyType": "url",
+  }
+</script>
```

Comment:
I thought about doing it via data attributes (like @NeilvB has in his [WIP version](https://github.com/alphagov/static/commit/bf516ac88a368d5baa1b08b3761f6e3bdedfcbd0#diff-3d28a36beb5925e7c6261c07ad15301bR39), but figured they'd be quite unwieldy with the number of options possible.